include/decoration.h

```diff
@@ -1,19 +1,348 @@
+extern const struct SpritePalette gUnknown_083ECA5C;
+extern const struct SpritePalette gUnknown_083ECA64;
+extern const struct SpriteTemplate gSpriteTemplate_83EC93C;
+extern const struct SpriteTemplate gSpriteTemplate_83ECA88;
+
+extern const struct Decoration gDecorations[];
+extern struct UnkStruct_02038900 gUnknown_02038900;
+extern u8 *gUnknown_020388D0;
+extern u8 gUnknown_081A2F7B[];
+extern u8 gUnknown_081A2F8A[];
+extern struct OamData gUnknown_020391AC;
+
+extern u8 gUnknown_020388D4;
+extern u8 gUnknown_020388D5;
+extern u8 gUnknown_020388F2;
+extern u8 gUnknown_020388F3;
+extern u8 gUnknown_020388F4;
+extern u8 gUnknown_020388F5;
+extern u8 gUnknown_020388F6;
+extern u8 gUnknown_020388D6[16];
+extern u8 gUnknown_020388E6[12];
+extern u16 gSpecialVar_0x8004;
+extern u16 gSpecialVar_0x8005;
+extern u16 gSpecialVar_0x8006;
+extern u16 gSpecialVar_0x8007;
+extern u16 gUnknown_020391A4;
+extern u16 gUnknown_020391A6;
+extern u8 gUnknown_020391A8;
+extern u8 gUnknown_020391A9;
+extern u8 gUnknown_020391AA;
+extern u8 gUnknown_02039234;
+extern void (*gUnknown_0300485C)(void);
+extern const struct YesNoFuncTable gUnknown_083EC96C[];
+extern struct UnkStruct_020391B4 gUnknown_020391B4[16];
+
+extern const u16 gUnknown_083EC654[3];
+extern const struct DecorationInventory gDecorationInventories[];
+extern const u8 *const gUnknown_083EC5E4[];
+extern const u8 *const gUnknown_083EC624[];
+extern u8 gUnknown_020388F7[];
+extern const struct MenuAction2 gUnknown_083EC604[];
+extern const struct MenuAction3 gUnknown_083EC634[];
+extern const u8 gUnknown_083EC65A[];
+extern const u8 gUnknown_083EC97C[];
+extern const u8 gUnknown_083EC984[];
+extern void sub_80B3068(u8); // src/shop
+extern const struct YesNoFuncTable gUnknown_083EC95C;
+extern const struct YesNoFuncTable gUnknown_083EC964;
+extern const struct YesNoFuncTable gUnknown_083EC9CC;
+extern const struct YesNoFuncTable gUnknown_083EC9D4;
+extern const struct YesNoFuncTable gUnknown_083ECAA0;
+
+extern u8 sub_8134194(u8); // src/decoration_inventory
+extern bool8 sub_81341D4(void); // src/decoration_inventory
+extern void sub_8134104(u8); // src/decoration_inventory
+extern bool8 sub_807D770(void);
+extern void sub_80F944C(void); // src/menu_helpers
+extern void sub_80F9480(u8 *, u8); // src/menu_helpers
+extern void sub_80F94A4(u8, u8 *, u8, s16); // src/menu_helpers
+extern void sub_80F94F8(u8 *); // src/menu_helpers
+extern void sub_80F9520(u8 *, u8); // src/menu_helpers
+extern void DestroyVerticalScrollIndicator(u8); // src/menu_helpers
+extern void CreateVerticalScrollIndicators(u8, u8, u8); // src/menu_helpers
+extern void LoadScrollIndicatorPalette(void); // src/menu_helpers
+extern void DisplayItemMessageOnField(u8, const u8 *, TaskFunc, u16); // src/menu_helpers
+extern void BuyMenuFreeMemory(void); // src/menu_helpers
+extern void sub_8109DAC(u8); // src/trader
+extern void ReshowPlayerPC(u8); // src/player_pc
+
+void sub_80FE2B4(void);
```

Comment:
i don't know the distinction between static vs non-static, or whether it in fact matters for this project